modules/ROOT/pages/exportpdf.adoc

```diff
@@ -0,0 +1,59 @@
+= Export to PDF plugin
+:navtitle: Export to PDF
+:description: The Export to PDF feature lets you generate a PDF file directly from the editor.
```

Comment:
Suggestion if we want to avoid using `you`
```suggestion
:description: The Export to PDF feature provides the ability to generate a PDF file directly from the editor.
```